### DISCUSSION TITLE
API - add status code

### DISCUSSION THREAD

**Discussion by alyabolowich** [electricitymaps/electricitymaps-contrib#5857] [Q&A] [answered]
Hi 👋  

I am wondering if the status code could also be returned in the request? This would help developers catch errors stemming from the EM API. (I am using the free version of the API, maybe this exists in the paid already?).

I see the documentation contains the keys "status" and "message" when I implement a faulty request but no code.

Thank you!

**Comment by VIKTORVAV99** [comment, marked as answer]
Hi!

The https status code should be returned in the http header so it's already included. In Python with requests you would access it like:

```python
response = Session.get(URL)
response.status_code
```

How you access this might differ depending on the ecosystem you use.

**Reply by alyabolowich** [reply]
Thank you!
